documentation/assemblies/assembly-kafka-connect-external-configuration.adoc

```diff
@@ -11,14 +11,13 @@
 
 = Using external configuration and secrets
 
-Kafka Connect connectors are configured using an HTTP REST interface.
-The connector configuration is passed to Kafka Connect as part of an HTTP request and stored within Kafka itself.
+Connectors are created, reconfigured, and deleted using an HTTP REST interface unless `KafkaConnectors` are enabled for the Kafka Connect cluster (for more information, see xref:con-creating-managing-connectors-{context}[]). The connector configuration is passed to Kafka Connect as part of an HTTP request and stored within Kafka itself.
```

Comment:
Seems a bit weird. Why not just say _Connectors are created, reconfigured, and deleted using an HTTP REST interface or using `KafkaConnectors`_?

Reply:
Looks like I tied myself in knots a bit there.

Your version is great, but we could add "if enabled" in brackets.